Automattic/newspack-plugin
# chore: remove newsroom screen from setup wizard

### All Submissions:






### Changes proposed in this Pull Request:

Removes the Newsroom screen from the Setup Wizard.

Closes https://github.com/Automattic/newspack-plugin/issues/807

### How to test the changes in this Pull Request:

1. Step through the Setup Wizard. Observe Newsroom screen is gone and the rest works properly.

### Other information:
